magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/SwMethodDocCheck.java

```diff
@@ -26,15 +27,28 @@ public class SwMethodDocCheck extends MagikCheck {
     public static final String CHECK_KEY = "SwMethodDoc";
     private static final String MESSAGE = "No or invalid method doc: %s.";
 
+    private static final boolean DEFAULT_IGNORE_BLANK_METHOD_DOC = false;
     private static final String PARAMETER_REGEXP = "[ \t]?([\\p{Lu}\\d_?]+)[^\\p{Lu}\\d_?]?";
     private static final Pattern PARAMETER_PATTERN = Pattern.compile(PARAMETER_REGEXP);
 
+    /**
+     * Ignore blank method doc.
```

Comment:
Does this mean it ignores an empty method doc, for any method? Or only for a blank method?

Perhaps also use `allow blank method doc`, instead of `ignore ...`.

Reply:
It means it doesn’t raise an issue for the next constructions. There are no parameters, but the linter expects some documentation. Using this toggle, you can make this check only trigger on missing parameter documentation.

```
_method a.b
_endmethod
$

_method a.b
    a.do_something()
_endmethod
$

_method a.b()
_endmethod
$

_method a.b()
    a.do_something()
_endmethod
$
```